test/e2e/storage/volume_provisioning.go

```diff
@@ -640,6 +640,14 @@ var _ = SIGDescribe("Dynamic Provisioning", func() {
 				claimSize:    "2Gi",
 				expectedSize: "2Gi",
 			}
+			// gce or gke
+			if getDefaultPluginName() == "kubernetes.io/gce-pd" {
+				// using GB not GiB as e2e test unit since gce-pd returns GB,
+				// or expectedSize may be greater than claimSize.
+				test.claimSize = "2G"
+				test.expectedSize = "2G"
+			}
```

Comment:
Does this matter - as long claimSize is greater than expected size? So when requesting 2Gi, I would assume we would allocated 3GB on GCE.

Reply:
Then expected size has to be set to "3G", right? IMO the current approach is cleaner, and it matches the 1.10 version of the test